server/integration_tests/test_data/demo_climatetrace/query_1/chart_config.json

```diff
@@ -80,42 +80,9 @@
                       "showHighest": true
                     },
                     "statVarKey": [
-                      "Annual_Emissions_GreenhouseGas"
```

Comment:
Might need to add Annual_Emissions_GreenhouseGas to the dc/topic/GreenhouseGasEmissions

Reply:
Actually remove "annual" in all descriptions and change dc/topic/GreenhouseGasEmissions description to be "ghg emissions by source"